doc/guides/encoder_plugins.rst

```diff
@@ -0,0 +1,115 @@
+.. _guide_encoder_plugins:
+
+==========================
+Pixel Data Encoder Plugins
+==========================
+
+Plugin Requirements
+===================
+
+Each available pixel data encoder in *pydicom* corresponds directly to a
```

Comment:
Since this is in the Guides section, but seems quite technical, I think it could benefit from a note here pointing to the compressing tutorial for more general use/concepts.

Reply:
The guides are supposed to "... explain a subject at a fairly high level", but I'll add in a sentence with a link.